package.json

```diff
@@ -183,7 +183,7 @@
   },
   "scripts": {
     "prepublish": "gulp post-npm-install",
-    "postpublish": "bash -c \"git tag -a ${npm_package_version} -m \"${npm_package_version}\" && git push origin ${npm_package_version}\"",
+    "postpublish": "bash -c \"if [ -z \"$GITHUB_ACTION\" ]; then git tag -a ${npm_package_version} -m \"${npm_package_version}\" && git push origin ${npm_package_version}; else echo 'Running in GitHub Action. Must be tagged separately'; fi \"",
```

Comment:
What does "tagged separately" mean here?

Reply:
Do you think the postpublish script is now long enough that we should put it in a separate file? Then it will be more readable and we can add comments and stuff

Reply:
I've removed the message, which was confusing worded and printed into logs of the GitHub Action where it could be interpreted wrongly.